jupyter-on-gke/tests/test_hub.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import requests
+import yaml
+
+from packaging.version import Version as V
+
+
+def test_hub_up(hub_url):
+    r = requests.get(hub_url)
+    r.raise_for_status()
+    print("Jupyterhub up.")
+
+
+def test_api_root(hub_url):
+    """
+    Tests the hub api's root endpoint (/). The hub's version should be returned.
+
+    A typical jupyterhub logging response to this test:
+
+        [I 2019-09-25 12:03:12.051 JupyterHub log:174] 200 GET /hub/api (test@127.0.0.1) 9.57ms
+    """
+    r = requests.get(hub_url + "/hub/api")
+    r.raise_for_status()
+    info = r.json()
+    assert V("4") <= V(info["version"]) <= V("5")
```

Comment:
What's the expected value of `info["version"]` here? Is `V("4")` the major version?

Reply:
Yes 4 is the major version, the response is 
```
{'version': '4.0.2'}
```